apps/pwabuilder/src/app-index.ts

```diff
@@ -146,7 +196,7 @@ export class AppIndex extends LitElement {
         <div id="content">
           <div id="router-outlet"></div>
         </div>
-        <discord-box></discord-box>
+        ${this.pageName === "freeToken" || this.pageName === "congratulations" ? html`` : html`<discord-box></discord-box>`}
```

Comment:
we can return `null` here instead of html``. Technically its the "proper" way to do it from my understanding